README.md

```diff
@@ -570,23 +575,29 @@ auction or to a one level opening.
   nonforcing, and redouble is SOS showing a desire to run but without a clear
   suit preference.
 
-## After 2NT or 1♣️!-1♦️!-2NT
+## After 2NT, 1♣️!-1♦️!-2NT, or 1♣️!-1♦️!-2♥️!-2♠️!-2NT
```

Comment:
why doesn't the D and S render in the third bidding sequence?

Reply:
I'm not sure what you mean. Everything looks normal to me.

Reply:
![Screen Shot 2021-05-06 at 5 57 26 PM](https://user-images.githubusercontent.com/79194/117370897-0c9e6780-ae95-11eb-84e4-5892e558c349.png)

Reply:
I checked the bytes and the spade is

U+2660 BLACK SPADE SUIT character (&#x2660;)
U+FE0F VARIATION SELECTOR-16 character (&#xFE0F;)

So maybe your client is weirdly displaying the variation selector.

Reply:
still no good

Reply:
can you check the second diamond?

Reply:
Both diamonds are the same:
U+2666 BLACK DIAMOND SUIT character (&#x2666;)
U+FE0F VARIATION SELECTOR-16 character (&#xFE0F;)

(e299a6 efb88f in UTF8)

00007410: 6365 2e0a 0a23 2320 4166 7465 7220 324e  ce...## After 2N
00007420: 542c 2031 e299 a3ef b88f 212d 31e2 99a6  T, 1......!-1...
00007430: efb8 8f21 2d32 4e54 2c20 6f72 2031 e299  ...!-2NT, or 1..
00007440: a3ef b88f 212d 31e2 99a6 efb8 8f21 2d32  ....!-1......!-2
00007450: e299 a5ef b88f 212d 32e2 99a0 efb8 8f21  ......!-2......!
00007460: 2d32 4e54 0a0a 2a20 33e2 99a3 efb8 8f21  -2NT..* 3......!